File: CSE110 [Programming Language - I]/Lab/Lab Assignment-2 (Variables & Datatypes)/Python/Task7.py
distance = 1204
hours = 5
minutes = 56
seconds = 23
total_hours = hours + (minutes / 60) + (seconds / 3600)

distance_km = distance / 1000
distance_miles = distance / 1609

velocity_kmh = distance_km / total_hours
velocity_mph = distance_miles / total_hours

print("Your velocity in km/h is ", velocity_kmh)
print(f"Your velocity in km/h is {velocity_kmh:.7f}")
print(f"Your velocity in miles/h is {velocity_mph:.7f}")

File: CSE110/Lab/Lab Assignment-2 (Variables & Datatypes)/Java/Task7.java
public class Task7 {
    public static void main(String[] args) {
        double distance = 1204;
        double hrs = 5;
        double mins = 56;
        double seconds = 23;
        double totalhrs = hrs + (mins / 60) + (seconds / 3600);
        double distanceKm = distance / 1000;
        double distanceMiles = distance / 1609;
        double velocityKmh = distanceKm / totalhrs;
        double velocityMph = distanceMiles / totalhrs;

        System.out.println("Your velocity in km/h is " + velocityKmh);
        System.out.printf("Your velocity in km/h is %.7f%n", velocityKmh);
        System.out.printf("Your velocity in miles/h is %.7f%n", velocityMph);
    }
}
